Plugwise_Smile/Smile.py

```diff
@@ -145,25 +147,18 @@ async def _create_session() -> aiohttp.ClientSession:
         self.smile_version = ()
         self.notifications = {}
 
-    async def connect(self, retry=2):
+    async def connect(self):
         """Connect to Plugwise device."""
-        # pylint: disable=too-many-return-statements
-        url = f"{self._endpoint}{DOMAIN_OBJECTS}"
-        try:
-            with async_timeout.timeout(self._timeout):
-                resp = await self.websession.get(url, auth=self._auth, headers=self._headers)
-            if resp.status == 401:
-                raise self.InvalidAuthentication
-        except (asyncio.TimeoutError, aiohttp.ClientError):
```

Comment:
InvalidAuthentication and either TimeOut/ConnectionFailed must remain in here for proper handling and as per HA requirements for config_flow